_health-care/_js/components/personal-information/Gender.jsx

```diff
@@ -14,13 +20,6 @@ class Gender extends React.Component {
 
   handleChange() {
     const gender = this.refs.gender.value;
```

Comment:
I found this pattern instead
  https://github.com/department-of-veterans-affairs/vets-website/blob/master/_health-care/_js/components/form-elements/ErrorableTextInput.jsx#L27

This allows us to avoid needing to use refs. "string" based ref attributes are apparently deprecated.  Doing this lets you remove the ref attribute on line 36.

Also, I think you can remove the this.state setting in the constructor().